Dockerfile.devkit

```diff
@@ -95,12 +73,12 @@ RUN mkdir -p /usr/share/ansible/collections \
 
 # hadolint ignore=DL4006
 RUN  curl -Lf https://github.com/mesosphere/mindthegap/releases/download/v"${MINDTHEGAP_VERSION}"/mindthegap_v"${MINDTHEGAP_VERSION}"_linux_amd64.tar.gz |tar xzf - -C /usr/local/bin
-ARG GOSS_VERSION=v0.3.18
-RUN curl -L "https://github.com/aelsabbahy/goss/releases/download/${GOSS_VERSION}/goss-linux-amd64" -o /usr/local/bin/goss-amd64
-RUN curl -L "https://github.com/aelsabbahy/goss/releases/download/${GOSS_VERSION}/goss-linux-arm64" -o /usr/local/bin/goss-arm64
```

Comment:
I think we dont need arm64 goss binary as we dont build remote arm64 images. if this was added intentionally, please let me know. I will addd it back.

Reply:
I am not aware of usecases and places where goss runs locally. can you please provide more details?

Reply:
i am unsure as well. can we get @mkarroqe to test this branch and see if everything works as expected and we can squash this?

Reply:
sure give me a few to test this

Reply:
yep I was able to run the following on this branch with no issues:
```
make clean
make bin/konvoy-image-wrapper
```

was also able to successfully create an AMI
```
==> amazon-ebs.kib_image: Goss validate ran successfully
==> amazon-ebs.kib_image: 
==> amazon-ebs.kib_image: 
==> amazon-ebs.kib_image: 
==> amazon-ebs.kib_image: Downloading spec file and debug info
    amazon-ebs.kib_image: Downloading Goss specs from, /tmp/goss-spec.yaml and /tmp/debug-goss-spec.yaml to current dir
.
.
.
Build 'amazon-ebs.kib_image' finished after 15 minutes 14 seconds.
```

Reply:
cool thank you

Reply:
Thank you @mkarroqe and @faiq